app/Http/Controllers/UsersController.php

```diff
@@ -3,10 +3,16 @@
 namespace App\Http\Controllers;
 
 use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
+use App\Models\User;
+
 
 class UsersController extends Controller
 {
-    public function index(){
-        return view('user_list');
-    }
-}  
+    public function search(Request $request){
+
+        $users = User::all();
+        return view('user_list',['users' => $users]);
```

Comment:
ControllerからViewに変数を引き渡す時、変数名をそのままで行うのであればcompactというヘルパ関数を使うと可読性が上がります。
```suggestion
        return view('user_list', compact('users'));
```